<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddAllergenen</class>
 <widget class="QDialog" name="AddAllergenen">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>579</width>
    <height>653</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTableView" name="tvAllergenen">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>164</y>
     <width>571</width>
     <height>485</height>
    </rect>
   </property>
   <property name="alternatingRowColors">
    <bool>true</bool>
   </property>
   <property name="selectionBehavior">
    <enum>QAbstractItemView::SelectRows</enum>
   </property>
   <attribute name="verticalHeaderDefaultSectionSize">
    <number>42</number>
   </attribute>
   <attribute name="verticalHeaderMinimumSectionSize">
    <number>42</number>
   </attribute>
  </widget>
  <widget class="QFrame" name="frame_2">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>52</y>
     <width>571</width>
     <height>106</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>481</width>
     <height>106</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>106</height>
    </size>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
   <property name="lineWidth">
    <number>2</number>
   </property>
   <widget class="my_lbFoto" name="lbFoto">
    <property name="geometry">
     <rect>
      <x>465</x>
      <y>5</y>
      <width>101</width>
      <height>96</height>
     </rect>
    </property>
    <property name="acceptDrops">
     <bool>true</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(231, 231, 231);</string>
    </property>
    <property name="text">
     <string>img</string>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="lbBenaming">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>10</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Benaming:</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbOmschrijving">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>40</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Omschrijving:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="tbBenaming">
    <property name="geometry">
     <rect>
      <x>95</x>
      <y>5</y>
      <width>361</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>tbBenaming</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="tbOmschrijving">
    <property name="geometry">
     <rect>
      <x>95</x>
      <y>35</y>
      <width>361</width>
      <height>66</height>
     </rect>
    </property>
    <property name="plainText">
     <string>tbOmschrijving</string>
    </property>
   </widget>
  </widget>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>5</y>
     <width>571</width>
     <height>41</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>481</width>
     <height>41</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>41</height>
    </size>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <property name="lineWidth">
    <number>2</number>
   </property>
   <widget class="QPushButton" name="btnNieuw">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="../resource.qrc">
      <normaloff>:/icons/icons/add.png</normaloff>:/icons/icons/add.png</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="btnBewerk">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>0</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="../resource.qrc">
      <normaloff>:/icons/icons/edit_button.png</normaloff>:/icons/icons/edit_button.png</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="btnOpslaan">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>0</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="../resource.qrc">
      <normaloff>:/icons/icons/save_close.png</normaloff>:/icons/icons/save_close.png</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="btnStop">
    <property name="geometry">
     <rect>
      <x>525</x>
      <y>0</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="../resource.qrc">
      <normaloff>:/icons/icons/stop2.png</normaloff>:/icons/icons/stop2.png</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="btnLaadPicture">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>0</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="../resource.qrc">
      <normaloff>:/icons/icons/picture_add.png</normaloff>:/icons/icons/picture_add.png</iconset>
    </property>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>my_lbFoto</class>
   <extends>QLabel</extends>
   <header>my_lbfoto.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../resource.qrc"/>
 </resources>
 <connections/>
</ui>
